srcpkgs/WoeUSB/template

```diff
@@ -1,34 +1,23 @@
 # Template file for 'WoeUSB'
 pkgname=WoeUSB
-version=3.3.1
-revision=3
-build_style=gnu-configure
-configure_args="--with-wx-config=wx-config-gtk3"
-hostmakedepends="automake gettext libtool"
-makedepends="wxWidgets-gtk3-devel"
-depends="WoeUSB-cli"
+version=5.2.4
+revision=1
+depends="bash grub parted wget dosfstools ntfs-3g wimlib"
 short_desc="Create a Windows USB stick installer from a real Windows DVD or image"
-maintainer="bra1nwave <bra1nwave@protonmail.com>"
+maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-3.0-or-later"
-homepage="https://github.com/slacka/WoeUSB"
-distfiles="https://github.com/slacka/WoeUSB/archive/v${version}.tar.gz"
-checksum=0cab88a1113506f39d2f1c19532b2cd8d968c6a9f59129953c000e29e73f3d4f
+homepage="https://github.com/WoeUSB/WoeUSB"
+distfiles="https://github.com/WoeUSB/WoeUSB/archive/v${version}.tar.gz>${pkgname}-${version}.tar.gz"
+checksum=8305296d49b7b58c200f9d373eac8dd494d03442737183749a0ee166403a7e68
+replaces="WoeUSB-cli>=0"
```

Comment:
`replaces` is wrong here. Leave WoeUSB-cli as a transitional package depending on WoeUSB instead.

Reply:
Thank you for the review. Compliance.